src/lib/services/feature-toggle-service.ts

```diff
@@ -1257,6 +1257,30 @@ class FeatureToggleService {
         );
     }
 
+    async bulkUpdateEnabled(
+        project: string,
+        featureNames: string[],
+        environment: string,
+        enabled: boolean,
+        createdBy: string,
+        user?: User,
+        shouldActivateDisabledStrategies = false,
+    ): Promise<void> {
+        await Promise.all(
```

Comment:
I will test later if sequential creation is needed or if we can run it in parallel

Reply:
Also I will do error handling once I have the UI and transactions